debezium-connector-sqlserver/src/main/java/io/debezium/connector/sqlserver/SqlServerConnectorConfig.java

```diff
@@ -245,6 +247,24 @@ public static SnapshotIsolationMode parse(String value, String defaultValue) {
             .withDescription("The name of the database the connector should be monitoring. When working with a "
                     + "multi-tenant set-up, must be set to the CDB name.");
 
+    public static final Field SERVER_TIMEZONE = Field.create(DATABASE_CONFIG_PREFIX + SqlServerConnection.SERVER_TIMEZONE_PROP_NAME)
+            .withDisplayName("Server timezone")
+            .withType(Type.STRING)
+            .withImportance(Importance.LOW)
+            .withValidation((config, field, problems) -> {
+                String value = config.getString(field);
+                try {
+                    ZoneId.of(value, ZoneId.SHORT_IDS);
```

Comment:
@jvenant When value is not provided it ends up in NPE - seen in tests, please execute the check only for non null values.

Reply:
oops... Sorry about that :flushed: